src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/BaseConvertDataFunctionalTests.cs

```diff
@@ -229,6 +228,20 @@ public static IEnumerable<object[]> GetDataForJson()
             });
         }
 
+        public static IEnumerable<object[]> GetDataForFhirToHl7v2()
+        {
+            var data = new List<string[]>
+            {
+                new[] { @"ObservationBundle", @"ObservationBundle.json", @"ObservationBundle-expected.hl7" },
```

Comment:
Should there be a case for ExamplePatient.json that you added in sample data as well?

Reply:
It could be another example. In this PR the intent was to include the necessary FHIR resources to generate an ORU^R01 result HL7v2 message and the minimum needed for that would be a FHIR bundle of Patient and Observation, and Encounter (depending on HL7 version).